README.md

```diff
@@ -28,7 +28,27 @@ Role Variables
 - `datadog_apt_repo` - Override default Datadog `apt` repository
 - `datadog_apt_key_url` - Override default url to Datadog `apt` key
 - `datadog_apt_key_url_new` - Override default url to the new Datadog `apt` key (in the near future the `apt` repo will have to be checked against this new key instead of the current key)
-- `datadog_allow_agent_downgrade` - Set to `yes` to allow agent downgrades on apt-based platforms (use with caution, see `defaults/main.yml` for details)
+- `datadog_agent_allow_downgrade` - Set to `yes` to allow agent downgrades on apt-based platforms (use with caution, see `defaults/main.yml` for details)
+
+Agent 6 (beta)
+--------------
+
+To upgrade or install agent6, you need to:
+
+- set `datadog_agent6` to true
+- either set `datadog_agent_version` to an existing agent6 version
```

Comment:
for the Chef cookbook I chose to use a different a variable to define the agent 6 pinned version, in order to allow setting both the agent 5 and 6 version at a high level (typically at the infrastructure level) and then allow users to selectively install the agent 5 or 6 on given hosts by just flipping the `datadog_agent6` boolean to true or false.

I'm not sure the approach works as well here with Ansible, since the role doesn't handle other things that need to be updated when upgrading to Agent 6 (like the changes in the config options supported in `datadog.conf` vs `datadog.yaml`). So since the upgrade can't be made to be as simple as flipping a boolean, it might not make sense to use different version variables.

Anyway, just wanted to make sure you were aware of this, if you have any thoughts on this...

Reply:
I don't think users will wildly mix agent5 and agent6. They may activate agent6 on a few specific hosts. So I don't think it's going to be a problem. We might change it later if we got feedback on it.

What do you think ?

Reply:
hmm yeah makes sense, let's change this only if we get feedback on it 👍